server/scripts/deleteTwimlApp.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
+'use strict';
+require('dotenv').config({ path: __dirname + '/../.env' });
+const process = require('node:process');
```

Comment:
In the delete I do need this since I write the twiML app ID to the 'env' file in the create script. https://github.com/twilio/twilio-voice-react-native-app/pull/115/files#diff-567a10a72d4f9e157a3a80eb52bf6ce6728e93d96ead4faf62812dacaf9fd159R40